### ISSUE TITLE
Problem with addClassNames() in 0.4.0: Uncaught InvalidCharacterError

### ISSUE THREAD

**Issue by abrighton** [japgolly/scalacss#81]
When I upgrade scalacss from 0.3.1 to 0.4.0, I get this error in the javascript console:

`
Uncaught InvalidCharacterError: Failed to execute 'add' on 'DOMTokenList': The token provided ('Styles-layoutPanel panel panel-default') contains HTML space characters, which are not valid in tokens.
`

The code looks like this:

```
import scalacss.Defaults._

// CSS styles
object Styles extends StyleSheet.Inline {

  import dsl._addClassNames

  import language.postfixOps

  val layoutPanel = style(
    addClassNames("panel", "panel-default"),
    marginTop(100 px),
    minWidth(300 px),
    maxWidth(500 px)
  )
...

```

The browser points me to this code as the source of the error:

```
trait ScalatagsJsDomImplicits {

  implicit final def styleaToJsDomTag(s: StyleA): Modifier = new Modifier {
    def applyTo(t: dom.Element) = t.classList.add(s.htmlClass)
  }

```

I get the same results in various browsers (The code is running under Ubuntu-16.04).

**Comment by japgolly** [comment]
That was fixed in [0.4.1](https://github.com/japgolly/scalacss/blob/master/doc/history/0.4.md). It should go away after upgrading.

**Comment by japgolly** [comment]
Reopen if this is isn't fixed.
